images/runtime/dotnetcore/dotnetCoreVersions.txt

```diff
@@ -1,4 +1,4 @@
-1.0, microsoft/dotnet:1.0.14-runtime
-1.1, microsoft/dotnet:1.1.11-runtime
+1.0, mcr.microsoft.com/dotnet/core/runtime:1.0.15
+1.1, mcr.microsoft.com/dotnet/core/runtime:1.1.12
```

Comment:
These should also be `aspnet` since we are building an image for web apps to run which require aspnetcore binaries in it.